common/lib/xmodule/xmodule/course_module.py

```diff
@@ -506,9 +506,9 @@ class CourseFields(object):
     )
     cert_name_long = String(
         help=_(
-            "Use this setting only when generating PDF certificates. "
-            "Between quotation marks, enter the long name of the course to use on the certificate that students "
-            "receive when they complete the course."
+            'Use this setting only when generating PDF certificates. '
+            'Between quotation marks, enter the long name of the type of certificate that students '
+            'receive when they complete the course. For instance, "Certificate of Achievement".'
```

Comment:
I'd change "Certificate of Achievement" to "Verified Certificate" (we use "certificate of achievement" only sporadically).